CHANGELOG.md

```diff
@@ -13,11 +22,14 @@ These should not affect the functionality of the module.
 
 **Merged pull requests:**
 
+- release 3.0.0 [\#75](https://github.com/voxpupuli/puppet-logrotate/pull/75) ([bastelfreak](https://github.com/bastelfreak))
 - Fix changelog [\#73](https://github.com/voxpupuli/puppet-logrotate/pull/73) ([alexjfisher](https://github.com/alexjfisher))
 - Added support for dateyesterday within logrotate::rule [\#71](https://github.com/voxpupuli/puppet-logrotate/pull/71) ([cliff-wakefield](https://github.com/cliff-wakefield))
 - BREAKING: Introduce Puppet 4 datatypes and drop Puppet 3 support [\#68](https://github.com/voxpupuli/puppet-logrotate/pull/68) ([mmerfort](https://github.com/mmerfort))
+- release 2.0.0 [\#64](https://github.com/voxpupuli/puppet-logrotate/pull/64) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
isn't this fine where it was before?

Reply:
I think one of the latest commits on github-changelog-generator orders PRs different. I didn't do this by hand.